src/framework/ui/support/components/checkmark/checkmark.component.tsx

```diff
@@ -58,24 +59,24 @@ export class CheckMark extends React.Component<CheckMarkProps> {
 
     return (
       <Component style={[container, styles.container]}>
-        <Component style={[shape, left, styles.shape, styles.left]}/>
-        <Component style={[shape, right, styles.shape, styles.right]}/>
+        <Component style={[shape, left, styles.shape, styles.left]} />
+        <Component style={[shape, right, styles.shape, styles.right]} />
       </Component>
     );
   }
 }
 
 const styles = StyleSheet.create({
   container: {
-    transform: [{ rotate: '-5deg' }],
+    transform: [{ rotate: I18nManager.isRTL ? '5deg' : '-5deg' }, { rotateY: I18nManager.isRTL ? '180deg' : '0deg' }],
```

Comment:
this is bad, it would be better to actually make a checkmark for the RTL instead of swapping the two element and rotate 180 degree.

Reply:
Refactored